src/invertColor.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+/**
+ * Write a function accepting 6 digit HEX color with a leading #
+ * and returning inverted color
+ *
+ * invertColor('#000000') === '#FFFFFF'
+ * invertColor('#DDEEAA') === '#221155'
+ * invertColor('#012345') === '#FEDCBA'
```

Comment:
Do we want this function to support shorthand colors too?

Reply:
I think we could start with a simple version (just)